src/sentry/api/endpoints/organization_eventid.py

```diff
@@ -60,7 +60,7 @@ def get(self, request, organization, event_id):
                     "organizationSlug": organization.slug,
                     "projectSlug": project_slugs_by_id[event.project_id],
                     "groupId": six.text_type(event.group_id),
-                    "eventId": six.text_type(event.id),
```

Comment:
Correct me if I am wrong, this should already be event_id since when I fetch an event from Snuba the implementation of the id method is:

```
@property
    def id(self):
        # Because a snuba event will never have a django row id, just return
        # the hex event_id here. We should be moving to a world where we never
        # have to reference the row id anyway.
        return self.event_id
```

Reply:
Yes, id is the same as event_id for a SnubaEvent (but not an Event). Wouldn't be an issue anywhere when we fetch the event from Snuba (like here), however it would be an issue during post process. I updated every instance of event.id to event.event_id in Sentry for consistency.

Reply:
To be clearer, this PR doesn't change any actual behaviour for users. We weren't actually using event.id in post process anyway. (The immediate issue was that some of the tests were wrong and started failing when I tried disabling saving). The other non test changes are just for consistency.

Reply:
Why would it be an issue in post processing ? If we serialize the event from a snuba event, we have event_id = id, if we fetch it again from snuba we are fine.

Reply:
Well, we avoid hitting Snuba during post process, we have a Django Event at that point.

Reply:
right, we schedule the task with the event we created without re-fetching it from snuba.